sphinx/source/docs/user_guide/annotations.rst

```diff
@@ -153,15 +152,15 @@ information and examples.
 .. note::
     :ref:`userguide_interaction_legends` features currently work on the first,
     "per-glyph" style legends. Legends that are created by specifying a column
-    to automatically group do no yet support interactive features.
+    to automatically group do not yet support interactive features.
 
 .. _userguide_plotting_color_bars:
 
 Color Bars
 ----------
 
 A |ColorBar| can be created using a |ColorMapper| instance, which
-contains a color palette. Both on- and off-plot color bars are
+contains a color palette. Both on-plot and off-plot color bars are
```

Comment:
Similiar to the commtents below, I understand Google style guide to be OK with leaving out the base for all except the last modifier in a situation like this (https://developers.google.com/style/hyphens#suspended-hyphens).

Reply:
@tcmetzger Can you elaborate on this may be with an example. _"Please change headlines to sentence case (capitalize words like in a regular sentence, but do not use any punctuation at the end)."_ Thanks

Reply:
Absolutely! This is best explained in the Google Style guide: https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings 
So, instead of writing "This Headline is Important", you should write "This headline is important" (only capitalizing the first letter and proper nouns).